### DISCUSSION TITLE
error crosstab & subsetting

### DISCUSSION THREAD

**Discussion by AngelaSherina** [teamalgoritma/community#307] [Q&A] [answered]
error saat melakukan subsetting untuk menjawab no 3 pada bagian bawah coding.
mohon solvingnya. terima kasih
![image](https://user-images.githubusercontent.com/112871237/190624397-3fc1ba56-09d9-4a3d-8ae8-41ccd2a7ca02.png)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dead @AngelaSherina 

Error yang Anda alami disebabkan oleh kesalahan syntax. Jika Anda ingin melakukan multi-index dataframe pada `crosstab` seharusnya masukan paramater `index` atau `columns` dibalut menggunakan `list` atau tanda `[ ]`.
Jika diperbaiki maka kodenya akan seperti ini:
```
pd.crosstab(index = [online_bl['month'], online_bl['sub_category']],
            columns = online_bl['scale'])
```
Jika diperhatikan, masukan parameter index dibalut dengan tanda `[ ]` karena memiliki lebih dari 1 argument masukan.
Kemudian, untuk menjawab soal nomor 3, diperhatikan juga bahwa yang ingin dicari adalah rata-rata harga terendah. Sehingga jangan lupa untuk memasukkan parameter `values` dan juga `aggfunc` pada fungsi crosstab dengan nilai yang sesuai

_Hint: untuk values adalah kolom harga untuk aggfunc adalah rata-rata (mean)_

Semoga dapat membantu. Apabila jawaban ini membantu Anda, bisa berikan "Mark as answer" pada comment ini.
